dependencies.gradle

```diff
@@ -26,7 +26,7 @@ ext.plugins = [
     "com.adarshr:gradle-test-logger-plugin:3.2.0",
     "com.github.ben-manes:gradle-versions-plugin:0.44.0",
     "com.netflix.nebula:gradle-lint-plugin:17.8.0",
-    "org.openapitools:openapi-generator-gradle-plugin:6.0.1",
```

Comment:
Our Templates and customized AbstractJavaJAXRSServerCodegen have to be updated as well while preserving changes we made.

Also version 6.3 was released a few days ago so we should probably update to that version while we are at it.

Reply:
AbstractJavaJAXRSServerCodegen was updated when we went to 6.0. I don't believe it needs updating. Let me know if you see something.